public/netsuke/docs/getting-started/index.html

```
@@ -210,7 +208,7 @@ <h2 class="font-serif text-2xl font-bold text-charcoal mb-4 group cursor-pointer
                 <div class="relative group mb-6">
                     <div class="absolute top-3 right-3 opacity-0 group-hover:opacity-100 transition-opacity">
                         <button class="bg-charcoal-light/20 hover:bg-charcoal-light/40 text-stone-light p-1.5 rounded transition-colors" title="Copy to clipboard">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)